### ISSUE TITLE
Icon's color in status bar

### ISSUE THREAD

**Issue by mahmoud-eslami** [time-mastery/pmodoro-application#10]
## Issue Description
The icons for the app status (e.g. network, battery, etc.) do not change color when the app's theme is changed. For example, when the app's theme is changed to dark mode, the status icons remain black, which causes them to disappear against the dark background at the top of the screen.

<table>
  <tr>
    <td>
      <img src="https://raw.githubusercontent.com/mahmoud-eslami/resume/hide-contribute/pmodoro/issues/light_bug.jpg" alt="light issue" width=200>
    </td>
    <td>
      <img src="https://raw.githubusercontent.com/mahmoud-eslami/resume/hide-contribute/pmodoro/issues/dark_bug.jpg" alt="dark issue" width=200>
    </td>
  </tr>
</table>

## Expected Behavior
When the app's theme is changed, the color of the status icons should also change to ensure they are visible against the new background color.

<table>
  <tr>
    <td>
      <img src="https://raw.githubusercontent.com/mahmoud-eslami/resume/hide-contribute/pmodoro/issues/dark_fixed.jpg" alt="dark fixed" width=200>
    </td>
  </tr>
</table>

**Comment by gantanikhilraj** [comment]
I can work on this, assign it to me.

**Comment by gantanikhilraj** [comment]
@mahmoud-eslami 
I would like to provide an update regarding the issue at hand. Upon conducting tests, I have observed that the functionality is performing well. Specifically, it successfully modifies the color of the status bar icons to correspond with the current mode, whether it be light or dark. To aid in conveying my point, I have included accompanying images for your reference.

![94c91dae-5e42-4acb-a577-d7a390c5a31e](https://user-images.githubusercontent.com/112814295/236499575-4d9e4f71-0a2d-407b-8312-8328007a9df9.jpg)
![b041f030-a92c-4cfd-a6e4-b38fb97813ac](https://user-images.githubusercontent.com/112814295/236499584-f5f22e38-6141-494f-afd9-605743c8f9ee.jpg)
